src/test/java/org/jenkinsci/plugins/githubautostatus/StatsdNotifierConfigTest.java

```diff
@@ -53,9 +53,7 @@
 public class StatsdNotifierConfigTest {
 
     private BuildStatusConfig config;
-    private final String repositoryOwner = "mock-pwner";
-    private final String repository = "mock-repo";
-    private final String branch = "mock-branch";
+    private final String id = "mock-id";
```

Comment:
Is this the externalized ID? Looks like it's in a different format if so.

Reply:
Updated name and made the mock format more complex